packages/@lwc/errors/src/compiler/error-info/template-transform.ts

```diff
@@ -625,4 +625,12 @@ export const ParserDiagnostics = {
         level: DiagnosticLevel.Warning,
         url: '',
     },
+    INVALID_TEMPLATE_WARNING: {
+        code: 1153,
+        message:
+            'Non-root template elements must contain valid LWC template directives, otherwise, the template and its children will be ignored. ' +
```

Comment:
```suggestion
            'Non-root template elements must contain valid LWC template directives. Otherwise, the template and its children will be ignored. ' +
```

Reply:
Also wondering if the wording can be tweaked further to add some clarity. Something like:

> Non-root template elements must contain one and only one attribute, and it must be a valid LWC template directive. If unexpected attributes are encountered, the template and its children will be ignored.

I know it's a nit pick, but "contain" makes me initially think you're talking about a child element of that template.

Reply:
Great suggestion @divmain!  I updated the wording to specify the LWC directives should be attributes on the template element.